[1-click/trusted] documentation. Significant update to 1-click solution page (#155)
VerifiedInc/VerifiedInc.github.io: docs/1-click-signup.mdx

            C->>+V: Call GET /sharedCredentials with sharedCredentialsUuid
            V->>-C: Return user data
        end
        opt
            note left of C: 4. (optional) Call Update 1-Click Credentials
            U-->>+C: Updates or enriches provided credential data
            C-->>V: Call /1-click/{uuid}/credentials with the user's updated data
            loop Asynchronously every time the user uses the updated credential values
                U-->>V: Uses the updated credential values
                V-->>C: Earn revenue share
            end
        end
    end
```

> *Please note that same as in the hosted flow, Verified Inc. will still sending the SMS on your behalf in Sandbox, however the same [go live](/#go-live) checklist applies and you will need handle the SMS in Production.*

Reviewer: ```suggestion
*Please note that, just as in the hosted flow, Verified Inc. will send the SMS on your behalf in sandbox, but you will need send the SMS in production (the same [go live](/#go-live) checklist applies).*
```

## Trusted Variant

Status: *Beta*

> The [`/1-click/trusted`](/api-reference#trusted-1-click) flow is in many ways similar to the [non-hosted](/#non-hosted-variant) flow documented above, however does not require your brand to use the OTP provided by the `/1-click` non-hosted response body. This variant is most appropriate for organizations that already use a phone authentication solution. As the name implies, it requires additional verification by Verified Inc. during the integration's development to ensure that phone possession is checked, often via a SMS one-time-passcode, during the current user session prior to calling this endpoint.

Reviewer: ```suggestion
The [`/1-click/trusted`](/api-reference#trusted-1-click) flow is in many ways similar to the [non-hosted](/#non-hosted-variant) flow documented above, but it does not require your brand to use the OTP provided by the `/1-click` non-hosted response body. This variant is most appropriate for organizations that already use a phone authentication solution. As the name implies, it requires additional verification by Verified Inc. during the integration's development to ensure that phone possession is checked, often via a SMS one-time passcode, during the current user session prior to calling this endpoint.
```

The notable differences between the trusted and non-hosted variants are:

1. **The endpoint is `/1-click/trusted` not `/1-click`.** This endpoint's request body resembles `/1-click`'s but the response body does not.

    <OneClickCredentialsResponseBodyDetails />

> 2. **1-click credential data is return synchronously from this endpoint.**

Reviewer: ```suggestion
2. **1-click credential data is return synchronously by this endpoint.**
```

>     In the alternative `/1-click` flows the need for the OTP to be provided makes for the user's data to be passed back asynchronously via methods documented in the [receive user data](/#receive-user-data) section above.

Reviewer: ```suggestion
    In the alternative `/1-click` flow, you need to send the OTP back to use once you've received it from the user, and we pass back the user's data asynchronously via methods documented in the [receive user data](/#receive-user-data) section above.
```

> If interested in this mode of integration please let us know and we will enable it for your API key. Please note, that we will have to validate your integration in Sandbox prior to sharing Production access to this feature. 

Reviewer: ```suggestion
If you're interested in this type of integration, please let us know and we will enable it for your API key. Please note that we will have to validate your integration in sandbox prior to sharing production access to this feature. 
```
